packages/template-retail-react-app/app/pages/page-viewer/utils.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {Region} from '../../components/experience/region'
+
+export const componentMapProxy = new Proxy(
+    {},
+    {
+        // eslint-disable-next-line no-unused-vars
+        get(_target, prop) {
+            let componentClass
+            switch (prop) {
+                default:
+                    componentClass = (props) => (
+                        <div style={{marginBottom: '10px'}}>
```

Comment:
I'm guessing all this code will be removed before merging.

Any idea why we see a style warning in the browser console?
```
Did not expect server HTML to contain the text node ".css-niwogb{color:var(--chakra-colors-black);}" in <select>
```